tutendero/src/main/kotlin/com/tutendero/api/security/jwt/CustomUserDetailsService.kt

```diff
@@ -0,0 +1,16 @@
+package com.tutendero.api.security.jwt
+
+import com.tutendero.api.repository.UserRepository
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.security.core.userdetails.UserDetailsService
+import org.springframework.security.core.userdetails.UsernameNotFoundException
+import org.springframework.stereotype.Component
+
+@Component
```

Comment:
this is a service, please use that spring steretype, if we want to implement an advice relate to this layer, It can avoid it